README.md

```diff
@@ -107,8 +107,8 @@ $ bash <(curl -fsSL https://raw.githubusercontent.com/liuchengxu/space-vim/maste
 #### Makefile
 
 ```bash
-$ git clone https://github.com/liuchengxu/space-vim.git ~/.space-vim
-$ cd ~/.space-vim
+$ git clone https://github.com/liuchengxu/space-vim.git /path/to/space-vim
```

Comment:
I'm thinking about if it could be better for the consistency favor to use a environment variable for space-vim location at all the places, like

```bash
$ export SPACE_VIM_LOCATION=/custom-path
$ git clone https://github.com/liuchengxu/space-vim.git "$SPACE_VIM_LOCATION/space-vim"

```

Reply:
Oh I apologize I didn't respond yet while I thought about it. While an environmental variable could be used in the installation directions I don't think it necessary.

I'd seen the /path/to/git-repository installation pattern used in [this](https://github.com/ingydotnet/git-subrepo) project:
```bash
git clone https://github.com/ingydotnet/git-subrepo /path/to/git-subrepo
echo 'source /path/to/git-subrepo/.rc' >> ~/.bashrc
```

Interesting and simple pattern.